HelpSource/Classes/Server.schelp

```diff
@@ -58,11 +58,30 @@ argument:: addr
 an optional instance of link::Classes/NetAddr::, providing IP address of the remote machine and port the app is listening on.
 
 argument:: options
-an optional instance of link::Classes/ServerOptions::. If code::nil::, an instance of ServerOptions will be created, using the default values.
+an optional instance of link::Classes/ServerOptions::. If code::nil::, an instance of ServerOptions will be created, using the default values. warning::To enable remote connections you will need to change the option link::Classes/ServerOptions#-bindAddress:: as the default value only allows connections from the local machine. code::s.options.bindAddress = "0.0.0.0":: will allow connections from any address.::
 
 argument:: clientID
 an integer. In multi-client situations, every client can be given separate ranges for link::Classes/Node##Nodes::, link::Classes/Buffer##Buffers::, or link::Classes/Bus##Busses::. In normal usage, the server will supply an ID automatically when a client registers for the link::#-notify#notifications:: so you should not need to supply one here. N.B. In multi-client situations, you will need to set the link::Classes/ServerOptions#-maxLogins:: to at least the number of clients you wish to allow. This must be the same in the Server instances on every client.
 
+code::
+// example usage:
+// on machine running the server
+(
+s.options.bindAddress = "0.0.0.0"; // allow connections from any address
+s.options.maxLogins = 2; // set to correct number of clients
+s.boot;
+)
+
+// on remote machine connecting to server
+(
+o = ServerOptions.new;
+o.options.maxLogins = 2;
+t = Server.remote(\remote, NetAddr("192.168.0.130", 57110), o); // set to correct address and port
+// info about returned client ID should be posted in the post window
+t.makeWindow; // make a window for monitoring
+)
```

Comment:
I'd prefer `var options` and `~remoteServer` over the `a-z` globals. Using full words improves readability.